Listen to mousedown and keydown only when safe

howyay opened the pull request:
Listen to mousedown and keydown only when editable text boxes are not focused (helps cursor problem to a point (#1)) EDIT: This is meant to refresh the rendering when user tries to click off/unfocus a text box which could imply that the user is now done editing and wants to see the updated rendering.
Update KaTeX to latest version available;
Allow double dollar sign delimiters (which is the norm?).

Commit:
Listen to mousedown and keydown only when safe

Commit:
Add render upon navigation between pages

Penguinlay commented:
I will review and update the code for more optimization this weekend. Thanks for the PR!

howyay commented:
I must inform you of a mistake I have made: the navigation detection part detects click events but the user action listener listens to mousedown, so it would cause the script to render twice when navigating between pages in notion. I addressed this issue in the latest commit in my fork, should I submit a PR for that? 
[235b6af260fc9c30209dd960e1969fb8e3fbf03e](https://github.com/howyay/notion-inline-latex/commit/235b6af260fc9c30209dd960e1969fb8e3fbf03e)

EDIT: forgot to mention that I also added a event listener for window.onload so the startup rendering doesn't start too early, which caused some issues in one of my math notes(which had lots and lots of stuff to load)

EDIT2: sorry for the bad writeup and formatting
